agent/util-scripts/pbench-tool-meister-client

```diff
@@ -68,9 +68,20 @@ def main(argv):
             # properly shut down.
             return 0
 
+    redis_server = os.environ.get("PBENCH_REDIS_SERVER", f"localhost:{def_redis_port}")
+    parts = redis_server.split(":", 1)
+    if len(parts) != 2:
+        raise Exception("Bad Redis server specified, '{redis_server}'")
+    try:
+        redis_port = int(parts[1])
+    except Exception:
+        raise Exception("Bad Redis port specified, '{redis_server}'")
```

Comment:
Let's see if exception string changes will be acceptable.

Reply:
The code is reporting the unparsed "redis server" argument, but it's objecting here to just the port; thus, I was thinking that it might provide better insight if we reported specifically what the code thought the port value was, rather than the unparsed input.